src/BloomBrowserUI/bookEdit/js/readerTools.ts

```diff
@@ -357,10 +358,17 @@ function resizeWordList(): void {
     if (synphony.source) {
 
       var height = Math.floor(currentHeight - top);
-      if (synphony.source.useAllowedWords === 1)
-        height += 20;
-      else
+      if (synphony.source.useAllowedWords === 1) {
```

Comment:
Deserves a comment. I gather you're trying to adjust the height based on whether space is needed to show the message about the list being truncated. Are constant numbers good enough? Won't be messed up by any kind of zooming or localization using different fonts?